### DISCUSSION TITLE
Allow source editing

### DISCUSSION THREAD

**Discussion by sorvis** [outline/outline#5941] [Ideas]
It would be nice to be able to edit the source of wiki pages. Sometimes edits will get in weird states such as not being able to remove a new line between bullet points or something like that. Or perhaps I want to make mass edits to formatting changes like indenting some bullet points or cleaning up a document that was imported as a word doc--if it was easy to edit the source then I could do a find and replace in a text editor.

**Comment by alkanna** [comment]
I agree having a raw markdown edit mode could be very useful to work around things that may not be polished enough.

**Comment by lgrahl** [comment]
This feature is dearly missed. I'm at least 2x faster when editing Markdown texts over WYSIWYG because WYSIWYG always comes with extremely wonky cursor, insertion, deletion and positioning behaviour.

**Comment by problem** [comment]
This is covered in #3326
